Documentation for position, rotation, scaling (#13)
cgi-studio-gmbh/incari-doc: toolbox/incari/object/get-alpha.md

# Overview

![The Get Alpha Node.](../../../.gitbook/assets/toolbox/incari/object/get-alpha.PNG)

> | Input | Output |

Reviewer: Lines 5-8 can be deleted.

| :--- | :--- |
| Object ID | Alpha |

> **Get Alpha** returns the *alpha* value of the color space of an **Object** in a **Scene**. The **Object ID** is given as input to the **Node**. 

Reviewer: I would maybe mention that this node only works with objects that have the "Sprite" category of attributes (List, Image Sequence, Web Sprite, Sprite, Text)

> 

Reviewer: # Attributes

|Attribute|Type|Description|
|---|---|---|
|`Object`|**ObjectID**|The **Object** that you wish to return the `alpha` of, if one isn't provided in the  `Object ID` **Socket**.|

# Overview

![The Get Alpha Node.](../../../.gitbook/assets/toolbox/incari/object/get-alpha.PNG)

**Get Alpha** returns the *alpha* value of the color space of an **Object** in a **Scene**. The **Object ID** is given as input to the **Node**. 

The **Get Alpha** node only works with objects that have **Sprite** category as an attribute such as **List**, **Image**, **Sequence**, **Web Sprite**, **Sprite** and **Text**.

# Attributes

|Attribute|Type|Description|
|---|---|---|

> |`Object`|**ObjectID**|The **Object** that you wish to return the `alpha` component of, if one is not provided in the `object ID` **Socket**.

Reviewer: ```suggestion
|`Object`|**ObjectID**|The **Object** whose `Alpha` **Attribute** value you wish to return if one is not provided in the `Object ID` **Socket**.
```
